slotfills/sections/repeatable.tsx

```diff
@@ -11,21 +11,54 @@ import AddMoreButton from '../../components/add-more-button';
 
 import './repeatable.scss';
 
-interface ChildProps {
+const CustomKnob = React.forwardRef<HTMLDivElement, {}>((props, ref) => (
+  <span ref={ref} className="fm-gutenberg-move-handle" aria-label={__('Move', 'fm-gutenberg')}>::</span>
```

Comment:
Was the `::` intentional here?

Reply:
yeah, it's my temporary "handle" for dragging things.